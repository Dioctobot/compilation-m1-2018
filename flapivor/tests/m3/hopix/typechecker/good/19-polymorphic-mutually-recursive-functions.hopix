def false_rec : forall a, b. a -> b (x) = false_rec2<b, a> (x)

and false_rec2 : forall a, b. b -> a (x) = false_rec<b, a> (x)

val main : String = false_rec<Int, String> (43)
