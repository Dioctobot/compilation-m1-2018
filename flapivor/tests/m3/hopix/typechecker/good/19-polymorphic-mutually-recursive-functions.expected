false_rec : forall a, b. a -> b
false_rec2 : forall b, a. a -> b
main : String
